winres/winres.json

```diff
@@ -0,0 +1,54 @@
+{
+  "RT_GROUP_ICON": {
+    "OTHER": {
+      "0000": "hazelcast-icon-flat-sm.png"
+    }
+  },
+  "RT_MANIFEST": {
+    "#1": {
+      "0409": {
+        "identity": {
+          "name": "",
+          "version": ""
+        },
+        "description": "",
+        "minimum-os": "win7",
+        "execution-level": "as invoker",
```

Comment:
What does `as invoker` do?

Reply:
I'm not sure about the possible values but I think this is where you specify if this executable is intended to work like a service or invocation base

Reply:
It seems `execution-level` can have three values: https://github.com/tc-hib/winres/blob/e87828d02c266dfb1edac897bf51090a9ae44971/manifest.go#L80